TestFoundation/TestNSTextCheckingResult.swift

```diff
@@ -18,7 +18,7 @@
 #endif
 
 class TestNSTextCheckingResult: XCTestCase {
-    static var allTests: [(String, TestNSTextCheckingResult -> () throws -> Void)] {
+    static var allTests: [(String, (TestNSTextCheckingResult) -> () throws -> Void)] {
```

Comment:
Oops, sorry I forgot to mention that! Without the parentheses, there's the following warning:

"Single argument function types require parentheses"